cf/test/test_Data.py

```diff
@@ -977,103 +977,98 @@ def test_Data_cached_arithmetic_units(self):
         # Reset
         cf.constants.CONSTANTS["FM_THRESHOLD"] = fmt
 
-    @unittest.skipIf(TEST_DASKIFIED_ONLY, "no attribute '_auxiliary_mask'")
-    def test_Data_AUXILIARY_MASK(self):
+    def test_Data_concatenate(self):
         if self.test_only and inspect.stack()[0][3] not in self.test_only:
```

Comment:
We can nuke these "test_only" lines now that the tests are so fast.

Reply:
Can I nuke them all (there's OOM 10-100 still here in the `test_Data` module) in a separate PR along with the LAMA-only methods, e.g. the above, instead of here, to keep the PR nice and self-contained?

Reply:
Sure - makes sense (sorry for not thinking of that!)

Reply:
Not at all, it's good to raise issues as you spot them :) I've noted to do a follow-on PR (after I have put up the `stats` and `concatenate_data` PRs).